### PR TITLE
CT-1757 add case type to close page

### PR DESCRIPTION
**Pull request by beccapearce**

### CODE REVIEW

File: app/views/cases/sar/_close_form.html.slim [hunk 1]
```diff
@@ -3,7 +3,7 @@
   .form-group
     = f.gov_uk_date_field :date_responded, { legend_text: t('.close_date'),
             form_hint_text: t('.date_example'),
-            today_button: {class: 'button-secondary'} }
+            today_button: {class: 'acts-like-button'} }
```
Comment (aliuk2012) [inline app/views/cases/sar/_close_form.html.slim]: why are we using this css class?

File: app/views/cases/foi/_close_form.html.slim [hunk 2]
```diff
@@ -5,7 +5,10 @@
 - url = defined?(form_url) ? form_url : process_closure_case_path(id: @case.id)
 = form_for @case, as: :case_foi, url: url do |f|
   .form-group
-    = f.gov_uk_date_field :date_responded, { legend_text: t('.close_date'), form_hint_text: t('.date_example') }
+    = f.gov_uk_date_field :date_responded, { legend_text: t('.close_date'),
+            form_hint_text: t('.date_example'),
+            today_button: {class: 'acts-like-button'} }
```
Comment (aliuk2012) [inline app/views/cases/foi/_close_form.html.slim]: why are we using this css class?

File: app/views/cases/sar/_close_form.html.slim (hunk 1 above)
Comment (beccapearce) [reply app/views/cases/sar/_close_form.html.slim]: I needs to be a link not a button
Happy to change it to something else

Comment (aliuk2012) [reply app/views/cases/sar/_close_form.html.slim]: You could try just an empty string.

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**